app/helpers/template_helpers.py

```diff
@@ -15,7 +15,7 @@
 
 
 @lru_cache(maxsize=None)
-def get_page_header_context(language, theme):
+def get_page_header_context(language: str, theme: str) -> dict:
```

Comment:
`Mapping` should be used rather than `dict`.
```suggestion
def get_page_header_context(language: str, theme: str) -> Mapping:
```
- `Mapping` will also need to be imported
- This applies to all places where `dict` is used in type hints